src/client/components/Stock.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+
+export default class Stock extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      stock: []
+    }
+  }
+  componentDidMount() {
+    fetch('http://localhost:3000/api/stock', {
```

Comment:
daj ścieżkę względną czyli "/api/......” bo jak odpalimy to na innej domenie to trzeba bedzie zmieniac kod